Android/app/src/main/java/io/github/project_travel_mate/destinations/funfacts/FunFactsActivity.java

```diff
@@ -38,33 +39,32 @@
  */
 public class FunFactsActivity extends AppCompatActivity implements FunFactsView {
 
-    @BindView(R.id.vp)
-    ViewPager viewPager;
-    @BindView(R.id.animation_view)
-    LottieAnimationView animationView;
-
     private City mCity;
     private String mToken;
     private Handler mHandler;
     Boolean isFirstVisitEnd = true;
     public int scrollState;
 
-
+    @BindView(R.id.vp)
+    ViewPager viewPager;
+    @BindView(R.id.animation_view)
+    LottieAnimationView animationView;
+    @Nullable
```

Comment:
why this?

Reply:
`Caused by: java.lang.IllegalStateException: Required view 'toolbar' with ID 2131362437 for field 'toolbar' was not found. If this view is optional add '@Nullable' (fields) or '@Optional' (methods) annotation.`
App crashes when opening the FunFactsActivity without this being labeled nullable.

I took a look at the XML file and ran built the app in master, and there is actually no toolbar on this page. is this something that was broken in a previous pull request or always the case? if latter I can just remove toolbar altogether from this page.